### PR TITLE
Doc Edit

### PR DESCRIPTION
**Pull request by alai97**
Update to be reflected in https://github.com/DataDog/documentation/pull/21501/.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,7 +44,9 @@ jobs:
     steps:
       - checkout
       - datadog-static-analysis/analyze:
-          service: my-service
+          service: "my-service"
+          env: "ci"
+          site: datadoghq.com
```
Comment (maycmlee) [inline README.md]: Does this need to change with site region?

**Review by maycmlee — APPROVED** [review]
Approved, but left a questions about site region!

File: README.md (hunk 1 above)
Comment (alai97) [reply README.md]:
```suggestion
          site: {{< region-param key="dd_site" code="true" >}}
```

Comment (alai97) [reply README.md]:
```suggestion
          site: {{< region-param key="dd_site" code="true" >}}
```

**Review by modernplumbing — APPROVED** [review]

**Pull request merged**